### DISCUSSION TITLE
Zeek collecting data, not getting into Kibana

### DISCUSSION THREAD

**Discussion by pktmterenceg** [Security-Onion-Solutions/securityonion#3078] [Unsupported Versions]
Hi all,

Again, I just want to start by thanking everyone who contributes to this effort. I really appreciate it.

I have experienced a few hiccups in my (standalone) SO install, but it looks like Zeek is back to work, collecting data/creating logs as expected. 

However, none of this data appears to make it into Kibana. 

Where would I start looking, please, to diagnose this issue?

Thank you!

**Comment by weslambert** [comment]
You'll want to ensure that:

- Filebeat is running on the applicable node and able to connect to Logstash on the manager and feed it data from Zeek
- Logstash is running on the manager and able to receive data from Filebeat. 
- Redis is running on the manager, and the queue is not climbing or stagnant
- Logstash on the search node(s) can connect to Logstash on the manager and pull records from Redsi
- ES is running on the search node(s) and are not refusing data from Logstash for one reason or another.
